pkg/analysis/nomaps/testdata/src/a/a.go

```diff
@@ -0,0 +1,34 @@
+package a
+
+type (
+	DefinedType  map[string]Component
+	DefinedType2 *map[string]Component
+	DefinedType3 map[string]int
+)
+
+type (
+	TypeAlias  = map[string]Component
+	TypeAlias2 = map[string]*Component
+	TypeAlias3 = DefinedType
+	TypeAlias4 = *DefinedType
+)
```

Comment:
I think it would be preferable to follow the same pattern we have use in other places for the names where we describe the type and whether it is pointer or not
```suggestion
type (
	MapStringComponent  map[string]Component
	PtrMapStringComponent *map[string]Component
	MapStringInt map[string]int
)

type (
	MapStringComponentAlias?  = map[string]Component <- What's different between this and the first MapStringComponent? 
	MapStringPtrComponent = map[string]*Component
	MapStringComponentAlias = MapStringComponent
	MapStringComponentPtrAlias = *MapStringComponent
)
```

Reply:
```
	MapStringComponentAlias?  = map[string]Component <- What's different between this and the first MapStringComponent? 
```

MapStringComponent is defined type that we define, this is type alias. So that is different type, but underlying type is same.

Reply:
Ahh gotcha, ok